test/e2e/benchmark.js

```diff
@@ -79,12 +79,17 @@ async function profilePageLoad(pages, numSamples, retries) {
     if (runResults.some((result) => result.navigation.lenth > 1)) {
       throw new Error(`Multiple navigations not supported`);
     } else if (
-      runResults.some((result) => result.navigation[0].type !== 'navigate')
+      runResults.some(
+        (result) =>
+          result.navigation[0] && result.navigation[0].type !== 'navigate',
```

Comment:
Hmm, this shouldn't be empty. This suggests that this benchmark is broken.

Reply:
Good point. I don't recall why those conditions were there.

Agreed that throwing a more explicit error would be helpful. If we don't have at least one navigation entry, it either means that the [1-second pause on this line](https://github.com/MetaMask/metamask-extension/blob/53acfe271d6ab70f93a7efaf302f6e7750db30f0/test/e2e/benchmark.js#L31) isn't long enough for the page to finish loading, or that something is stopping us from using the Navigation API (e.g. LavaMoat scuttling).

Reply:
that's a v good callout @Gudahtt  and thank you @legobeat for asking in Slack too. 
The results for this PR are mostly `NaN` which seems to be broken now. 

![image](https://github.com/MetaMask/metamask-extension/assets/54408225/97095a4b-a55d-43e1-9ea0-c52f59bc63aa)